autogen/agentchat/conversable_agent.py

```diff
@@ -679,8 +680,9 @@ async def a_send(
             )
 
     def _print_received_message(self, message: Union[Dict, str], sender: Agent):
+        iostream = IOStream.get_default()
```

Comment:
I think we might not want to use the default stream, as the printings here are basically logging output -- serving similar purpose as logging.info. We should be using the console stream instead of default stream for replacing existing `print` statements.

Reply:
I am wondering if we should allow streams to be set in the constructor of `ConversableAgent`, or as a parameter in `initiate_chat`. Right now I see the same stream used for different purposes: logging, LLM response streaming, control messages like human input, speaker selection, etc.

Reply:
I am not sure that I understand. This function prints out the conversation between agents, it is needed on the client side.

Reply:
Resolved after office hour discussion.